sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_enums.py

```diff
@@ -0,0 +1,77 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from enum import Enum
+
+
+class KeyVaultRoleScope(str, Enum):
+    """Collection of well known role scopes. This list is not exhaustive"""
+
+    global_value = "/"  #: use this if you want role assignments to apply to everything on the resource
+
+    keys_value = "/keys"  #: use this if you want role assignments to apply to all keys
+
+
+class DataActionPermission(str, Enum):
+    """Supported permissions for data actions.
+    """
+
+    #: Read HSM key metadata.
```

Comment:
Make sure these names align with what @christothes defined in the swagger PR (not merged just yet but should be soon).